app/views/govuk_publishing_components/components/_google_tag_manager_script.html.erb

```diff
@@ -1,14 +1,19 @@
 <%
   gtm_auth ||= nil
   gtm_preview ||= nil
+
+  gtm_attributes = %w( gtm_cookies_win=x )
+  gtm_attributes << "gtm_auth="+gtm_auth if gtm_auth
+  gtm_attributes << "gtm_preview="+gtm_preview if gtm_preview
```

Comment:
This falls a bit off our linting rules - which I guess means we don't lint erb files 😞

switching to:

```
gtm_attributes = %w(gtm_cookies_win=x)
  gtm_attributes << "gtm_auth=" + gtm_auth if gtm_auth
  gtm_attributes << "gtm_preview=" + gtm_preview if gtm_preview
```
would sort it.

Reply:
Updated. Is there a place where I can find these lint rules so I can set them in my code editor?